### PR TITLE
py-hjson: new port

### PR DESCRIPTION
**Pull request by bashu**
#### Description

Hjson for Python

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.2 21G320 x86_64
Xcode 14.1 14B47b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-hjson/files/py27-hjson [hunk 1]
```diff
@@ -0,0 +1 @@
+bin/hjson-2.7
```
Comment (reneeotten) [inline python/py-hjson/files/py27-hjson:1]: remove this unused select file

File: python/py-hjson/files/py34-hjson [hunk 2]
```diff
@@ -0,0 +1 @@
+bin/hjson-3.4
```
Comment (reneeotten) [inline python/py-hjson/files/py34-hjson:1]: remove this unused select file

File: python/py-hjson/files/py35-hjson [hunk 3]
```diff
@@ -0,0 +1 @@
+bin/hjson-3.5
```
Comment (reneeotten) [inline python/py-hjson/files/py35-hjson:1]: remove this unused select file

File: python/py-hjson/files/py36-hjson [hunk 4]
```diff
@@ -0,0 +1 @@
+bin/hjson-3.6
```
Comment (reneeotten) [inline python/py-hjson/files/py36-hjson:1]: remove this unused select file

File: python/py-hjson/files/setup.py.patch [hunk 5]
```diff
@@ -0,0 +1,19 @@
+--- setup.py.orig	2022-12-15 18:37:31.000000000 +0600
++++ setup.py	2022-12-15 19:52:13.000000000 +0600
+@@ -70,14 +70,14 @@
+         author_email="laktak@cdak.net",
+         url="http://github.com/hjson/hjson-py",
+         license="MIT License",
+-        packages=['hjson', 'hjson.tests'],
++        packages=['hjson'],
```
Comment (reneeotten) [inline python/py-hjson/files/setup.py.patch:8]: why not install the tests?

File: python/py-hjson/Portfile [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+name                py-hjson
+version             3.1.0
+revision            0
+
+categories-append   devel
+
+license             MIT
+supported_archs     noarch
+
```
Comment (reneeotten) [inline python/py-hjson/Portfile:15]: check for correct setting of "platforms" line

**Review by reneeotten — CHANGES_REQUESTED** [review]
is this intended to be used as a library by other Python packages or is it a "program that is written in Python"? In the latter case we wouldn't need the other subports and the name could just be "hsjon" with a specified Python version and no need for the use of "select".

**Comment by bashu** [comment]
> is this intended to be used as a library by other Python packages or is it a "program that is written in Python"?

Yes, this is a library and intended to be used by other python packages. hjson_select was removed too.

**Comment by reneeotten** [comment]
thanks @bashu ; I'll merge this now. If in the future a "select" port is deemed useful that can be added.

**Comment by bashu** [comment]
> If in the future a "select" port is deemed useful that can be added.

Absolutely

**Pull request merged**